File: src/main/java/org/lasque/twsdkvideo/video_beauty/views/cosmetic/OnItemClickListener.java
package org.lasque.twsdkvideo.video_beauty.views.cosmetic;

import androidx.recyclerview.widget.RecyclerView;

/**
 * TuSDK
 * org.lasque.tusdkdemohelper.tusdk.newUI
 * qiniu-PLDroidMediaStreamingDemo
 *
 * @author H.ys
 * @Date 2020/8/5  14:31
 * @Copyright (c) 2020 tw. All rights reserved.
 */
public interface OnItemClickListener<I,H extends RecyclerView.ViewHolder> {
    void onItemClick(int pos, H holder, I item);
}

File: src/main/java/org/lasque/twsdkvideo/video_beauty/effectcamera/base/OnItemClickListener.kt
package org.lasque.twsdkvideo.video_beauty.effectcamera.base

import androidx.recyclerview.widget.RecyclerView


interface OnItemClickListener<I,H : RecyclerView.ViewHolder> {
    fun onItemClick(pos: Int, holder: H, item: I)
}

interface OnItemDeleteClickListener<I,H : RecyclerView.ViewHolder>{
    fun onItemDelete(pos: Int, holder: H, item: I)
}